.readthedocs.yml

```diff
@@ -0,0 +1,12 @@
+###############################################################
+# ReadTheDocs Configuration                                   #
```

Comment:
@yangjunqingqiao if the rebase has been done, why is this still in the files changed list?

Reply:
Ah, good catch I must’ve rebased wrt to a version of main before your packaging PR was merged. I’ll update it real fast.